ConceptMap/ConceptMap-Genomics-specimen-primarysample.json

```diff
@@ -0,0 +1,309 @@
+          "target": [
+            {
+              "code": "258450006",
+              "display": "Cerebrospinal fluid specimen (specimen)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "cord-blood",
+          "display": "Cord Blood",
+          "target": [
+            {
+              "code": "122556008",
+              "display": "Cord blood specimen (specimen)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "chorionic-villus-sampling",
+          "display": "Chorionic Villus Sampling",
+          "target": [
+            {
+              "code": "258565009",
+              "display": "Chorionic villi specimen",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "fetal-blood",
+          "display": "Fetal Blood",
+          "target": [
+            {
+              "code": "737357006",
+              "display": "Fetal blood specimen",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "fibroblasts",
+          "display": "Fibroblasts",
+          "target": [
+            {
+              "code": "119333000",
+              "display": "Fibroblast specimen",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "heart",
+          "display": "Heart",
+          "target": [
+            {
+              "code": "127462005",
+              "display": "Specimen from heart (specimen)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "kidney",
+          "display": "Kidney",
+          "target": [
+            {
+              "code": "127473003",
+              "display": "Specimen from kidney (specimen)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "liver",
+          "display": "Liver",
+          "target": [
+            {
+              "code": "119383005",
+              "display": "Specimen from liver (specimen)",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "lymph-node",
+          "display": "Lymph Node",
+          "target": [
+            {
+              "code": "258589002",
+              "display": "Lymph node specimen",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "other",
+          "display": "Other",
+          "target": [
+            {
+              "code": "123038009",
+              "display": "Specimen (specimen)",
+              "equivalence": "wider",
+              "comment": "Any other suitable descendant of 123038009|Specimen (specimen) not listed on this code. This should be provided by free text."
```

Comment:
maybe update to "This should be provided by free text **if no suitable SNOMED CT concept is found**"?